src/internal/cluster/tunnel.go

```diff
@@ -44,6 +44,7 @@ const (
 	ZarfLogging  = "LOGGING"
 	ZarfGit      = "GIT"
 	ZarfInjector = "INJECTOR"
+	Prometheus   = "PROMETHEUS"
```

Comment:
We shouldn't need these if we add our own service to connect to prometheus

Reply:
fixed thanks